backend/src/impl/benchmark_configs/config_miniglue.json

```diff
@@ -5,7 +5,7 @@
   "datasets": [
     {"dataset_name": "sst2", "metrics": [{"name": "F1", "weight": 0.2, "default": 0.0},
                                          {"name": "Accuracy", "weight": 0.8, "default":  0.0}]},
-    {"dataset_name": "snli", "metrics": ["Accuracy"]}
+    {"dataset_name": "snli", "metrics": [{"name": "Accuracy"}]}
```

Comment:
Will the code automatically perform `weighted sum` based on different types of datasets? (Since only sst2 needs to be aggregated)?

Reply:
Yes, this is done automatically because the the default weight of a metric is `1/len(metrics)`, which in this case is 1.

Reply:
I see.
Also, I want to confirm one thing:
when we aggregate over the `metric` dimension, `grouping by dataset` will be a necessary step right?(since we assume that different metrics within the same `dataset` could be aggregated flexibly combined with predefined weights)
I guess this assumption will encounter a problem when 'metric' is subdataset-dependent?
Maybe we can introduce more fine-grained `group-by` for example, [dataset_name, task] or [dataset_name, subdataset_name]

Reply:
Thanks! That was actually already supported, see here for an example: https://github.com/neulab/explainaboard_web/pull/201/files#diff-7b3652084a046cfb4db169bd6def69533be117be2ac8904da2ea82b65cae2317R20